.github/workflows/generate_formulae.brew.sh_data.yml

```
@@ -9,37 +9,41 @@ jobs:
   generate:
     if: github.repository == 'Homebrew/homebrew-core'
     runs-on: macos-latest
+    env:
+      HOMEBREW_COLOR: 1
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_AUTO_UPDATE: 1
     steps:
-      - uses: actions/checkout@master
-      - uses: Homebrew/actions/git-ssh@master
+      - name: Set up git-ssh
+        uses: Homebrew/actions/git-ssh@master
         with:
           git_user: BrewTestBot
           git_email: homebrew-test-bot@lists.sfconservancy.org
           key: ${{ secrets.FORMULAE_DEPLOY_KEY }}
-      - run: |
-          brew update-reset "$(brew --repository)"
-          brew tap "$GITHUB_REPOSITORY"
-          brew update-reset "$(brew --repository "$GITHUB_REPOSITORY")"
 
-          export PATH="$(brew --repo)/Library/Homebrew/vendor/portable-ruby/current/bin:$PATH"
-
-          git clone --depth=1 git@github.com:Homebrew/formulae.brew.sh
+      - name: Set up Homebrew
+        run: |
+          brew update-reset $(brew --repository)
+          brew tap ${{ github.repository }}
+          brew update-reset $(brew --repository ${{ github.repository }})
+          echo ${{ secrets.ANALYTICS_JSON_KEY }} > ~/.homebrew_analytics.json
+
+      - name: Build and push formulae.brew.sh
+        run: |
+          # clone formulae.brew.sh with SSH so we can push back
+          git clone --depth=1 --quiet git@github.com:Homebrew/formulae.brew.sh
           cd formulae.brew.sh
 
-          # setup analytics
-          echo "$ANALYTICS_JSON_KEY" > ~/.homebrew_analytics.json
-
           # run rake (without a rake binary)
+          export PATH="$(brew --repository)/Library/Homebrew/vendor/portable-ruby/current/bin:$PATH"
           ruby -e "load Gem.bin_path('rake', 'rake')"
```

I think this can now be:

```suggestion
          /usr/bin/rake
```

`rake` alone probably works - though it'll likely use whatever other version of Ruby is installed on the images rather than system rake.